admin/webpack.admin.config.js

```diff
@@ -0,0 +1,52 @@
+var webpack = require('webpack');
+var path = require('path');
+var common = require('../webpack.common.config.js');
+var assign = require('object-assign');
+var BundleTracker = require('webpack-bundle-tracker');
+
+var websiteRoot = path.join(__dirname, 'websiteRoot', 'static');
+
+var adminRoot = path.join(__dirname, 'static');
+
+var staticWebsitePath = function(dir) {
+    return path.join(websiteRoot, dir);
+};
+
+var staticAdminPath = function(dir) {
+    return path.join(adminRoot, dir);
+};
+
+common.entry['admin-base-page'] = staticAdminPath('js/pages/base-page.js')
```

Comment:
In-place modifications of data structures should be avoided when possible. Instead of modifying `common` directly, you can make a copy of it with the changes you need. `assign` is good for this.

```js
module.exports = assign({}, common, {
    entry: {
        'admin-base-page': staticAdminPath(...)
    },
    // ...
}
```